script.js

```diff
@@ -1,49 +1,47 @@
 // HTML Selectors
-const table = document.getElementById('mainTable')
-const btn = document.getElementById('btn')
-const dateField = document.getElementById('dateForCheckingAvailability')
-const districtsField = document.getElementById('districtCodes')
+const table = document.getElementById('mainTable');
+const btn = document.getElementById('btn');
+const dateField = document.getElementById('dateForCheckingAvailability');
+const districtsField = document.getElementById('districtCodes');
 
 // Init variables
-let dateOfAvailability = `${new Date().getDate() + 1}-${
-  new Date().getUTCMonth() + 1
-}-${new Date().getFullYear()}`
-dateField.value = dateOfAvailability
-let inputDistrictCodes = districtsField.value.replace(/\s/g, '').split(',')
-let isFilteredResult = false
+let dateOfAvailability = `${new Date().getDate() + 1}-${new Date().getUTCMonth() + 1}-${new Date().getFullYear()}`;
+dateField.value = dateOfAvailability;
+let inputDistrictCodes = districtsField.value.replace(/\s/g, '').split(',');
```

Comment:
Add comment here for explaining what the regex is doing
`// /s is for replacing spaces in the input field, if at all inputted by the user in the District Codes input box`